src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let sum = 0;
+
+  if (days >= 7) {
+    sum += (days * price) - 50;
```

Comment:
GJ . But I guess you can make small changes. Why won't you make variable that'll give you total price insted of counting it every time?
It's not a loop, there is no need to add sum if you can just change it.